File: 2st/Yeonwoo/조건에 맞게 수열 변환하기1.py
def solution(arr):
    answer = []
    for i in arr:
        if i >= 50 and i%2==0:
            answer.append(i//2)
        elif i<50 and i%2==1:
            answer.append(i*2)
        else:
            answer.append(i)
    return answer

File: 조건에 맞게 수열 변환하기1.java
class Solution {
    public int[] solution(int[] arr) {
        int[] answer = new int [arr.length];
        for(int i=0;i<arr.length;i++){
            if(arr[i]<50){
                if(arr[i]%2==1){
                    answer[i]=arr[i]*2;
                }
                else{
                    answer[i]=arr[i];
                }
            }
            else if(arr[i]>=50){
                if(arr[i]%2==0){
                    answer[i]=arr[i]/2;
                }
                else{
                    answer[i]=arr[i];
                }
            }
            else{
                answer[i]=arr[i];
            }
        }
        
        return answer;
    }
}